epub33/core/index.html

```diff
@@ -8711,9 +8711,49 @@ <h2>Security and Privacy</h2>
 		<section id="app-overview-unsupported" class="appendix">
 			<h2>Unsupported Features</h2>
 
-			<p>This specification contains certain features the Working Group no longer recommends for use or that are
-				only retained for legacy support reasons. This section defines the meanings of the designations attached
-				to these features and their support expectations.</p>
+			<p>This specification contains certain features that are not yet fully supported in Reading Systems, that
+				the Working Group no longer recommends for use, or that are only retained for interoperability with EPUB
+				2 Reading Systems. This section defines the meanings of the designations attached to these features and
+				their support expectations.</p>
+
+			<section id="risky">
+				<h3>Risky Features</h3>
+
+				<p>A <strong>risky</strong> feature is a feature introduced prior to EPUB 3.3 but that the Working Group
+					has not been able to establish is implemented in two or more Reading Systems as required by <a
+						href="https://www.w3.org/Consortium/Process/">W3C Process</a>.</p>
+
+				<p>These features are considered important to retain despite this limitation because they are known to
+					be implemented by EPUB Creators (i.e., their deprecation would invalidate existing content) and/or
+					they are integral to the content model on which EPUB is built.</p>
+
+				<p>If this specification designates a feature as risky, the following hold true:</p>
+
+				<ul>
+					<li>
+						<p><a>EPUB Creators</a> MAY use the features as described.</p>
+					</li>
+					<li>
+						<p><a>Reading Systems</a> SHOULD support the feature as described.</p>
+					</li>
+				</ul>
+
+				<p>Validation tools SHOULD alert EPUB Creators to the presence of risky features when encountered in
+					EPUB Publications but MUST NOT treat their inclusion as a violation of the standard (i.e., not emit
+					errors or warnings).</p>
+
+				<div class="caution">
+					<p>Whether risky labels are removed or replaced by deprecation in a future version of the standard
+						cannot be determined at this time. EPUB Creators should strongly consider the interoperability
+						problems that may arise both now and in the future when using these features.</p>
+				</div>
+
+				<div class="note">
+					<p>The marking of features as risky is a one-time event to account for the different process under
+						which EPUB was developed prior to being brought into W3C. This label will not be used on
+						features in the future.</p>
+				</div>
```

Comment:
I just stopped for a moment reading this; are we sure about this? I _think_ it is all right, but I would really like the approval of the others of this specific note; I worry that we may tie our hands unnecessarily for the future.

Reply:
I was assuming this is a one-time thing we can do to grandfather the spec into W3C, but for anything going forward we should be clear that we are going to follow W3C process more officially.

It was a late addition after I wrote the description for how the class works, as I was worried that by adding this class we might provoke reactions that we're trying to create our own looser process for EPUB. I assume moving forward that we'll only have new features without support, in which case they would follow the standard at-risk/removal process for going through to REC.

Reply:
Although looking at the section again, the first paragraph does say that this is for features developed prior to EPUB 3.3. The note is probably redundant except for the note using the label again. I assume testing will ferret out all the features so it won't be added again in the future, but it may not be strictly necessary to say.

Reply:
I agree with the intention and I am all right with what is stated.

I actually did not realize it is a duplicate of the first paragraph, but it is... ie, it may be not be necessary to keep it.

Reply:
One thing I don't like looking at it again is that it seems to suggest we would have to remove the labels in the next revision. That may or may not be true.

I've tweaked the wording of the note so to clarify that it won't be applied to new features, as they will follow w3c process. That should complement the first paragraph better.